.github/workflows/links.yml

```diff
@@ -0,0 +1,21 @@
+name: Link Checker
+on:
+  push:
+    branches: [ main ]
```

Comment:
Do you want to update these and add "1.*" so it runs on all cut 1.y specific branches and the "1.x"?

Reply:
Updated this